bundles/wps/wps-client/src/main/java/org/orbisgis/wpsclient/impl/WpsClientImpl.java

```diff
@@ -181,8 +183,8 @@ public void activate(){
         jobStateListenerList = new ArrayList<>();
         jobMap = new HashMap<>();
         refreshAvailableScripts();
-
-        editorManager.addEditorFactory(new ProcessEditorFactory(editorManager, this));
+        editorFactory = new ProcessEditorFactory(editorManager, this);
+        editorManager.addEditorFactory(editorFactory);
```

Comment:
Like this service https://github.com/orbisgis/orbisgis/blob/master/bundles/sqlconsole/src/main/java/org/orbisgis/sqlconsole/SQLConsoleFactory.java